overlord/hookstate/hookmgr.go

```diff
@@ -70,7 +70,9 @@ type Handler interface {
 	Done() error
 
 	// Error is called if the hook encounters an error while running.
-	Error(err error) error
+	// The returned bool flag indicates if the original hook error should be
+	// ignored by hook manager.
+	Error(err error) (bool, error)
```

Comment:
we might want to use named return args here: `(ignoreHookError bool, err error)`

Reply:
Done